crates/miden-objects/src/account/mod.rs

```diff
@@ -20,7 +20,7 @@ pub use builder::AccountBuilder;
 pub mod code;
 pub use code::{AccountCode, procedure::AccountProcedureInfo};
 
-mod component;
+pub mod component;
 pub use component::{
     AccountComponent, AccountComponentMetadata, AccountComponentTemplate, FeltRepresentation,
     InitStorageData, MapEntry, MapRepresentation, PlaceholderTypeRequirement, StorageEntry,
```

Comment:
since we are re-exporting the full module - should we get rid of the individual exports? It is breaking change, but given that we released `miden-objects` just yesterday, I think this may be fine.

Reply:
I'd maybe do that in the next release, since I'm not sure to what extent downstream crates are relying on them (i.e. I don't know if these are brand new, certainly some of them were in the last release). This patch _is_ a bit redundant, but at least provides a path for migrating in preparation for subsequent releases.